<!DOCTYPE HTML>
<html>
<head>
<title>ZhaoLab - ShanghaiTech University</title>
<link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all">
<link href="css/style.css" rel="stylesheet" type="text/css" media="all" />
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
<link href="css/camera.css" rel="stylesheet" type="text/css" media="all" />
    <script type='text/javascript' src='js/jquery.min.js'></script>
    <script type='text/javascript' src='js/jquery.mobile.customized.min.js'></script>
    <script type='text/javascript' src='js/camera.min.js'></script> 
    
    <script>
		jQuery(function(){
			
			jQuery('#camera_wrap_1').camera({
				thumbnails: true
			});

			jQuery('#camera_wrap_2').camera({
				height: '400px',
				loader: 'bar',
				pagination: false,
				thumbnails: true
			});
		});
		</script>
		<!--start lightbox -->
<link rel="stylesheet" type="text/css" href="css/jquery.lightbox.css">
<script src="js/jquery.lightbox.js"></script>
<script>
  // Initiate Lightbox
  $(function() {
    $('.gallery a').lightbox(); 
  });
</script>
</head>
<body>
	<div class="container">
		<div class="main-header">
			<div class="inner-side">
				<div class="header">
					<div class="logo">
						<h1><a href="index.html"><span>ZhaoLab</span></a></h1>
						<h5><b>iHuman Institute - ShanghaiTech University</b></h5>
						<a href="research-CN.html"><h5 style="float:right;padding-right:6em"><b>中文</b></h5></a>
					</div>
					<div class="head-top">
						<div class="top-menu">
						<span class="menu"><img src="images/nav.png" alt=""/> </span>
							<ul>
								<li><a href="index.html"><span>Home</span></a></li>
								<li><a href="openings.html"><span>Openings</span></a></li>
								<li class="active"><a href="research.html"><span>Research</span></a></li>
								<li><a href="publications.html"><span>Publications</span></a></li>
								<li><a href="people.html"><span>Members</span></a></li>
								<li><a href="resource.html"><span>Resource</span></a></li>
							</ul>
						</div>
						<!-- script for menu -->
									
							 <script>
							 $("span.menu").click(function(){
							 $(".top-menu ul").slideToggle("slow" , function(){
							 });
							 });
							 </script>
						<!-- //script for menu -->

						<div class="clearfix"></div>
					</div>
				</div>
					<div class="about-grids">
						<div class="col-md-12 about-grid">
							<h4>“Structure without function is a corpse,  
							 function without structure is a ghost” </h4> 
							 <p></p>
						<img src="images/ssf.PNG" width=70%>
						<p>My research group aim to develop and apply methods to gain understanding of protein sequence-structure-function relationship. We develop and apply methods in blue to address this problem. We work on enzymes and transmembrane receptors. </p>						

						</div>

						<div class="col-md-12 about-grid">
						<p></p>	
						<p></p>							
		
						<h4> GPCR mechanism study & ligand discovery.</h4>
						<p></p>	
						<img src="images/activation.png" width=60%>
						<p></p>

							
							<p>For transmembrane receptors such as G protein-coupled receptors (GPCRs), we focus on developing methods to understand the key questions such as (a) universal activation mechanism, (b) biased signalling mechanism and (c) G protein selectivity mechanism.

							<p>						
							Based on our understanding of these mechanisms, we do structure- and mechanism-based ligand discovery, espeically  allosteric ligands and biased ligands.  </p>
							
							<p>Integrative ligand docking: many GPCRs have been intensively researched and have a lot of known ligands. However, where do these ligands bind and how do they bind are not known for most of the cases. We use docking, together with mutation data, SAR data, selectivity data and potentially many other data, to find binding sites and poses of known ligands of GPCRs. Once we get the accurate binding poses, we try to understand the mechanism of these molecules.</p>
							
							<p>We offer the following tools: 
							(1) allosteric pathway mapping, (2) single point mutations that stabilize either active or inactive state of GPCRs, and (3) structure- and mechanism-based allosteric and biased ligand discovery. 
													
							</p>

						</div>
						<div class="col-md-12 about-grid">
						<p></p>	
						<p></p>								
						<h4>Discover novel enzyme functions, novel metabolites (natural products) and novel pathways</h4>
						<p></p>	
						<img src="images/g-2-np.png" width=70%>
			
						    <p>The number of protein sequences increase explosively nowadays. However, at least 50% of sequences have their functions unknown, uncertain or incomplete. Traditional bioinformatics approaches use “annotation transfer” to assign known functions to new sequences, and hardly to discover any new functions. 
							
							<p>We focus on developing methods to discover novel enzyme function, novel metabolites (especially natural products) and novel pathways. </p>
							
							<p>Tools we have for enzyme function prediction:
							(1) Pathway docking
							(2) Sequence similarity network (SSN)
							(3) Genome neighborhood network (GNN)
							(4) Machine learning.</p>
							
											<p></p>		
	
						<div class="clearfix"></div>
					 </div>

				<div class="new-grids col-md-12">
				<div class="col-md-6 new-grid">
					<div class="new-text">
						<h5>Quick Links</h5>
						<ul>
							<li><a href="http://ihuman.shanghaitech.edu.cn/" target="_blank"> > iHuman Institute</a></li>
							<li><a href="http://www.shanghaitech.edu.cn/eng/" target="_blank"> > ShanghaiTech University</a></li>
						</ul>
					</div>
				</div>
				<div class="col-md-6 new-grid">	
					<div class="new-text">
						<h5>Contact Us</h5>
						<ul>
							<li><a> Address: 393 Middle Huaxia Road, Pudong, Shanghai, 201210</a></li>
							<li><a> Phone: (+86)-21-20685005</a></li>
							<li><a> Email: zhaosw(at)shanghaitech.edu.cn</a></li>
						</ul>
					</div>
					<div class="clearfix"></div>
				</div>
				<div class="clearfix"></div>
			</div>
				<!--start-footer-section-->
		<div class="footer-section">
			<div class="footer-top">
				<p>Copyright &copy; 2014-2018 Zhaosuwen Laboratory. STU All Rights Reserved. Designed by Zhaolab.</p>
			</div>
		</div>
	<!--end-footer-section-->
			</div>
		</div>
	</div>
 </body>
</html> 
